README.md

```diff
@@ -1,6 +1,8 @@
 # sonar-cayc-stats-plugin
 
-Demonstrates the value of Clean as You Code methodology
+Demonstrates the value of Clean as You Code methodology. 
+
+Download the JAR file [here](https://binaries.sonarsource.com/?prefix=Distribution/sonar-cayc-plugin/).
```

Comment:
Aren't we missing an opportunity to describe the compatibility of the plugin as well? SQ 8.9/9.9 and onwards.

Reply:
Can we consider:

1. JAR -> plugin JAR
2. Adding some instruction on how to use the JAR (placing it in the right folder and restarting the server)

Reply:
it's just below

Reply:
🤦‍♂️ That was hidden by Github in the PR review. My bad.